liualgotrader/data/data_base.py

```diff
@@ -23,5 +23,9 @@ def get_symbol_data(
         return pd.DataFrame()
 
     @abstractmethod
-    def get_symbols(self) -> List[Dict]:
+    def get_market_snapshot(self) -> List[Dict]:
+        return []
```

Comment:
ok, have done the change.
"consider symbols are properties w/ getter rather than a regular function" <<< I dont understand this part. Could you clarify?